### PR TITLE
Document instanceOptions

### PR DESCRIPTION
**Pull request by ngfreiter**
Document the `instanceOptions` added to the config in https://github.com/FHIR/sushi/pull/937.

### CODE REVIEW

File: content/docs/SUSHI/configuration/_index.md
Comment (jafeltra) [inline content/docs/SUSHI/configuration/_index.md:144]: I feel like I read the "key is either" as the key can be one of these options, but not both. I tried changing it slightly to imply you can have any of the (two) keys described. Feel free to take or leave this.

```suggestion
| instanceOptions | N/A | A `key: value` pair, where key can be `setMetaProfile` (options are `always`, `never`, `inline-only`, `standalone-only`) or `setId` (options are `always` or `standalone-only`). 
```

**Review by jafeltra — COMMENTED** [review]
This looks good to me! I had one minor suggestion below.

Also, should we wait until the next SUSHI release to merge this PR?

**Review by jafeltra — APPROVED** [review]
Lookin' good!

⚠️ Don't merge until we have a new SUSHI release ⚠️

**Review by julianxcarter — APPROVED** [review]
Looks good!

**Pull request merged**
